python_ta/checkers/structure_test_checker.py

```diff
@@ -0,0 +1,56 @@
+"""
+checker for using constant value in a conditional statement
+"""
+import astroid
+
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
+
+
+class UsingStructureTestChecker(BaseChecker):
+
+    __implements__ = IAstroidChecker
+
+    name = 'using-constants-test'
+
+    msgs = {'W7778': ('using a structure which returns a constant value in a conditional statement'
+                      , 'using-constants-test'
+                      , 'Conditional statements should depend on a variable not a constant value.'
```

Comment:
Note: I don't think you've pushed the latest version of this. Please do so, but also consider my comment about the wording of the other message, and review your message for this one as well.